### PR TITLE
Feature branch/submit clinical page

### PR DESCRIPTION
**Pull request by ciaranschutte**
- submit clinical page
- functionality coped over from `platform-ui`
- adjusted for TS fixes, gql changes and uikit Table changes

### CODE REVIEW

File: src/app/(post-login)/submission/program/[shortName]/clinical-submission/types.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+export type { ClinicalSubmissionQuery as GqlClinicalSubmissionQuery } from "@/__generated__/graphql";
+
+// export type ClinicalSubmission = ClinicalSubmissionQuery["clinicalSubmissions"];
```
Comment (demariadaniel) [inline src/app/(post-login)/submission/program/[shortName]/clinical-submission/types.ts:21]: Surely Types can be commented back in?

File: src/app/components/ThemeProvider.tsx [hunk 2]
```diff
@@ -95,8 +95,6 @@ export default function ThemeProvider({
               font-family: inherit;
             }
             .rt-table {
-              display: block;
-              width: 0;
             }
```
Comment (demariadaniel) [inline src/app/components/ThemeProvider.tsx:98]: Can probably remove the selector as well then?

File: src/app/(post-login)/submission/program/[shortName]/registration/page.tsx [hunk 3]
```diff
@@ -116,9 +117,21 @@ export default function Register({
     },
   );
 
+  const [uploadClinicalSubmission, mutationStatus] = useMutation(
+    UPLOAD_CLINICAL_SUBMISSION_MUTATION,
+    {
+      onCompleted: (d) => {
+        //setSelectedClinicalEntityType(defaultClinicalEntityType);
```
Comment (demariadaniel) [inline src/app/(post-login)/submission/program/[shortName]/registration/page.tsx:124]: Is this purposefully commented out? Wondering if we need to keep track of functionality that's tied to the API integration / not able to implement right now

**Review by demariadaniel — APPROVED** [review]
Couple questions but all in all looks good!

File: src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/ClearSubmissionButton.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  clearSubmission,
+  refetchSubmission,
+}) => {
+  const handleSubmissionClear = async () => {
+    const toaster = useToaster();
+    const commonToaster = useCommonToasters();
+    const { setGlobalLoading } = useGlobalLoader();
+
+    setGlobalLoading(true);
+    await sleep(0);
```
Comment (demariadaniel) [inline src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/ClearSubmissionButton.tsx:45]: That's a weird trick

Comment (ciaranschutte) [reply src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/ClearSubmissionButton.tsx:45]: Sure is. Any suggestions on an improvement?

File: src/app/(post-login)/submission/program/[shortName]/registration/page.tsx (hunk 3 above)
Comment (ciaranschutte) [reply src/app/(post-login)/submission/program/[shortName]/registration/page.tsx:124]: good catch ty

File: src/app/components/ThemeProvider.tsx (hunk 2 above)
Comment (ciaranschutte) [reply src/app/components/ThemeProvider.tsx:98]: good point yes

File: src/app/(post-login)/submission/program/[shortName]/clinical-submission/types.ts (hunk 1 above)
Comment (ciaranschutte) [reply src/app/(post-login)/submission/program/[shortName]/clinical-submission/types.ts:21]: indeed - ty

**Pull request merged**